### DISCUSSION TITLE
Question about FDR for AMCOMBC

### DISCUSSION THREAD

**Discussion by xnysound** [FrederickHuangLin/ANCOMBC#155] [Q&A]
Hi, 

For the ancombc function, I have set the p_adj_method = "fdr". Does this mean it uses the Benjamini-Hochberg procedure to control the FDR at 0.05?

Thank you! 

Best,
Xinyi

**Comment by FrederickHuangLin** [comment]
Hi @xnysound,

Yes, "BH" or its alias "fdr" represents the B-H procedure. Please refer to `?p.adjust` for details.

Best,
Huang
